docs/running-a-node/ledger-management.md

```diff
@@ -25,7 +43,7 @@ When starting a new node the ledger must be downloaded and kept updated in order
 
 ### Tuning options
 
-Depending on machine and networking resources, the bootstrap performance can be improved by updating the following [configuration](/running-a-node/configuration/) values:
+Depending on machine and networking resources, the bootstrap performance can be improved by updating the following [configuration](/running-a-node/configuration/) values in the [`config-node.toml`](../running-a-node/configuration.md#configuration-file-locations) file:
```

Comment:
Is `../running-a-node` working for website?

Reply:
I updated the README to include details about new link formats that are preferred so the built-in MKDocs link checker can work. Please review this: https://github.com/nanocurrency/nano-docs/pull/307